Examples/Lenses/AutoLenses.stencil

```diff
@@ -0,0 +1,38 @@
+infix operator *~: MultiplicationPrecedence
```

Comment:
In our project I separate the bound lens "Library Code" from the stencil template (right now I'm not checking in the generated code, just the templates). I can work on cleaning it up because it's kind of neat! 
This is one of those things that become possible because of Sourcery, for me, I couldn't imagine hand-writing lens code for my models.